index.bs

```diff
@@ -1,11 +1,11 @@
 <pre class=metadata>
-Title: Web Audio API
+Title: Web Audio API 1.1
 Shortname: webaudio
-Level: 1
+Level: 1.1
 Group: audiowg
-Status: ED
+Status: FPWD
 ED: https://webaudio.github.io/web-audio-api/
-TR: https://www.w3.org/TR/webaudio/
+TR: https://www.w3.org/TR/webaudio-11/
```

Comment:
I'm not sure how many Web Audio versions we're planning on, but should this perhaps be "webaudio-1-1" or "webaudio-1_1"?

Reply:
This URL is suggested by @svgeesus.